app/javascript/components/patient/Patient.js

```diff
@@ -68,665 +111,614 @@ class Patient extends React.Component {
-                    {!showPlannedTravel && <span className="none-text">None</span>}
-                    {showPlannedTravel && (
-                      <React.Fragment>
-                        <Row>
-                          <Col>
-                            <b>Type:</b> <span>{this.props.details.additional_planned_travel_type || '--'}</span>
-                          </Col>
-                        </Row>
-                        <Row>
-                          <Col>
-                            <b>Place:</b>{' '}
-                            <span>
-                              {this.props.details.additional_planned_travel_destination_country}
-                              {this.props.details.additional_planned_travel_destination_state}
-                              {!this.props.details.additional_planned_travel_destination_country &&
-                                !this.props.details.additional_planned_travel_destination_state &&
-                                '--'}
-                            </span>
-                          </Col>
-                        </Row>
-                        <Row>
-                          <Col>
-                            <b>Port Of Departure:</b> <span>{this.props.details.additional_planned_travel_port_of_departure || '--'}</span>
-                          </Col>
-                        </Row>
-                        <Row>
-                          <Col>
-                            <b>End Date:</b>{' '}
+                    {!(showArrivalSection || this.props.details.travel_related_notes) && <div className="none-text">None</div>}
+                    {showArrivalSection && (
+                      <Row>
+                        <Col md={12} lg={24} xl={12} className="item-group">
+                          <p className="subsection-title">Departed</p>
+                          <div>
+                            <b>Port of Origin:</b> <span>{this.props.details.port_of_origin || '--'}</span>
+                          </div>
+                          <div>
+                            <b>Date of Departure:</b>{' '}
                             <span>
-                              {this.props.details.additional_planned_travel_start_date
-                                ? moment(this.props.details.additional_planned_travel_start_date, 'YYYY-MM-DD').format('MM/DD/YYYY')
-                                : '--'}
+                              {this.props.details.date_of_departure ? moment(this.props.details.date_of_departure, 'YYYY-MM-DD').format('MM/DD/YYYY') : '--'}
                             </span>
-                          </Col>
-                        </Row>
-                        <Row>
-                          <Col>
-                            <b>Start Date:</b>{' '}
+                          </div>
+                          <div>
+                            <b>Carrier:</b> <span>{this.props.details.flight_or_vessel_carrier || '--'}</span>
```

Comment:
Previously the Carrier and the Flight or Vessel # was under the "Arrival" section - why was it moved to "Departure"?

Reply:
I think I might have introduced this in the responsive mockup - previously Carrier/Flight # was a separate row after the Arrival row, and I moved it into the Departed column based on the visual alignment. Should be easy to move back to separate div just before the Notes div. 
```
<div class="item-group">
  <div><b>Carrier:</b>...</div>
  <div><b>Flight...</div>
</div>
```

Reply:
Fixed in my latest commit:
![Screen Shot 2021-02-09 at 8 33 32 AM](https://user-images.githubusercontent.com/35042815/107371880-c793b080-6ab2-11eb-80f9-656184ac9f06.png)

Reply:
Should `Source of Report` be in the arrival column?

Reply:
I don't know - that'd be the question for the PO team.

Reply:
Removed this field for now since it was a new addition.  Will add in this ticket: https://tracker.codev.mitre.org/browse/SARAALERT-280

Reply:
Not seeing the change to remove it - did you push yet?

Reply:
It was in this commit: [f9bd682](https://github.com/SaraAlert/SaraAlert/pull/688/commits/f9bd682472928c97de8341dc8062470871fc617d)